### PR TITLE
LPS-131501 Update `@liferay/npm-scripts` to v44.0.1

### PR DESCRIPTION
**Pull request by wincent**
[Release notes](https://github.com/liferay/liferay-frontend-projects/releases/tag/npm-scripts%2Fv44.0.1).

Basically, brings in [an update to `@liferay/eslint-config`](https://github.com/liferay/liferay-frontend-projects/releases/tag/eslint-config%2Fv24.0.1) because we [want this fix](https://github.com/liferay/liferay-frontend-projects/pull/517) for [this issue](https://github.com/liferay/liferay-frontend-projects/issues/515) to unblock [this PR](https://github.com/liferay-frontend/liferay-portal/pull/992).

In short, we now handle an edge case that was affecting the `group-imports` rule.

### CODE REVIEW

**Comment by wincent** [comment]
ci:forward
